js/signup.js

```diff
@@ -0,0 +1,15 @@
+import { emailInput, passwordInput, passwordCheckInput, signBtn, eyeBtn } from "./seletors.js";
+import { signupEmailErrorMessage, passwordCheckErrorMessage, passwordErrorMessage, signupBtn, toggleVisibility } from "./functions.js";
+
+document.addEventListener('DOMContentLoaded', () => {
+  emailInput.addEventListener('focusout', signupEmailErrorMessage);
+  passwordInput.addEventListener('focusout', passwordErrorMessage);
+  passwordCheckInput.addEventListener('focusout', passwordCheckErrorMessage);
+  signBtn.addEventListener('click', signupBtn);
+  eyeBtn.addEventListener('click', toggleVisibility);
```

Comment:
[여기](https://github.com/codeit-bootcamp-frontend/2-Weekly-Mission/pull/159#discussion_r1391060888)에서 `querySelectorAll`로 눈 모양 버튼을 조회했다면 클릭 리스너도 모든 버튼에 대해 등록해줘야겠네요.  
```suggestion
eyeBtns.forEach(btn => {
  btn.addEventListener('click', toggleVisibility);
})
```